kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -151,7 +152,9 @@
     protected ObjectSubType financialObjectSubType;
     protected AssetAcquisitionType acquisitionType;
     protected ContractsAndGrantsAgency agency;
+    protected List<Note> boNotes = new ArrayList<Note>();

```

Comment:
> You'd have to talk to the BAs about that.

Hate to pull rank, but the onus to collect info for review questions rests upon the reviewee (I just double checked from this side). If you are stating that you're unable to ask the BAs yourself, please do let Julie and I understand why this is so. (To be clear, I can help assist if it comes to it, but the initial attempt and subsequent coordination should be initiated from your end.)